docs/content/preview/explore/observability/active-session-history.md

```diff
@@ -190,176 +190,274 @@ Note that this is not an ASH-specific view, but can be used to extract more info
+ 8ea1aa0f2e4749a4802986fefbf54d43 | System     | system         | transactions | \xd555              | \xe000            |    66
+ e70fd34078e84fef945f28fe6b004309 | System     | system         | transactions | \x8000              | \x8aaa            |    64
+ e66879054249434da01b3e7a314b983c | System     | system         | transactions | \x4000              | \x4aaa            |    63
+ 9d64f34797924998911922563d3ad0ec | System     | system         | transactions | \xa000              | \xaaaa            |    63
+ 9cf4fc4a834040df91987b26bf9831ed | System     | system         | transactions | \x4aaa              | \x5555            |    63
+ c2cfa997bf634637880137870cee25a1 | System     | system         | transactions | \xcaaa              | \xd555            |    61
+ d3bbd37828ab4225a68bf4dd54924138 | System     | system         | transactions | \x1555              | \x2000            |    56
+ b3bbaec3014f4f1aa01e04b42c5f40c7 | System     | system         | transactions | \x3555              | \x4000            |    56
+ 542c6f91ff6a403399ddf46e4d5f29bb | System     | system         | transactions | \x6000              | \x6aaa            |    56
+ 27780cde5a1b445d83ff83c865b9bea5 | System     | system         | transactions | \xc000              | \xcaaa            |    51
+ 4a64a9f25e414ce7a649be2e6d24c7ee | System     | system         | transactions | \x2aaa              | \x3555            |    46
+```
+
+You can see that only a single tablet is getting most of the requests, and you have the partition range of that particular tablet.
+
+### Distribution of the type of the wait events for each component
+
+Detect where most time is being spent to help understand if the issue is related to disk IO, network operations, waiting for a condition or lock, or intense CPU work.
+
+```sql
+SELECT
+    wait_event_component,
+    wait_event_type,
+    COUNT(*)
+FROM
+    yb_active_session_history
+GROUP BY
+    wait_event_component,
+    wait_event_type
+ORDER BY
+    wait_event_component,
+    count;
+```
+
+```output
+ wait_event_component | wait_event_type | count
+----------------------+-----------------+-------
+ TServer              | WaitOnCondition |    47
+ TServer              | Network         |   910
+ TServer              | Cpu             |  2665
+ TServer              | DiskIO          |  8193
+ YSQL                 | LWLock          |     1
+ YSQL                 | Cpu             |  1479
+ YSQL                 | Network         |  4575
+```
+
+### Detect which client/application is sending the most amount of queries
+
+Suppose you have a rogue application which is sending a lot of queries, you can identify the application using the following example query:
+
+```sql
+SELECT
+    client_node_ip,
+    COUNT(*)
+FROM
+    yb_active_session_history
+WHERE
+    client_node_ip IS NOT NULL
+GROUP BY
+    client_node_ip;
+```
+
+```output
+ client_node_ip  |  sum
```

Comment:
This one should have been count instead of sum, I have fixed the doc. Please update it here